graphistry/pygraphistry.py

```diff
@@ -609,7 +609,11 @@ def set_spanner_config(spanner_config: Optional[Union[Dict, str]] = None):
 
         if spanner_config is not None: 
             PyGraphistry._config["spanner"] = spanner_config 
-
+    
+    @staticmethod
+    def set_kusto_config(kusto_config):
+        if kusto_config is not None:
+            PyGraphistry._config["kusto"] = kusto_config
```

Comment:
Hmm this pattern is increasingly risky for Louie as assume single tenant, vs cascading off a dynamic plottable instance

Reply:
Maybe there is a way to allow louie to control lifecycle, like this:
```
client = graphistry.kusto_connect()
graphistry.kusto_query(..., client=client) # otherwise can default to the global one
```

Reply:
I think we're good now:

`Plottable` now has:

```python
    _spanner_config: Optional[SpannerConfig]
    _kusto_config: Optional[KustoConfig]
```

and `KustoGraph` does a cascade

```python
class KustoGraphContext:
    def __init__(self, config: Union[KustoConfig, None] = None):
        config = config or PyGraphistry._config.get("kusto")
```

Reply:
I do broadly agree about moving to clients, both for graphistry server and db servers. I wonder if we can move some of this stuff to type state (dunno if python supports), like `Plottable<KustoClient,SplunkClient, ...>`